helm/draughtsman-chart/templates/configmap.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   namespace: draughtsman
 data:
   config.yaml: |
+    service:
+      deployer:
+        provider: {{ .Values.Installation.V1.Provider.Kind }}
```

Comment:
`service.deployer` section already exists in the secret. It's ok. `viper` is smart enough to merge the secret and configmap into a config. Tested.

Reply:
Actually, credit where credit is due, it's `microkit` who's smart about multiple config files https://github.com/giantswarm/microkit/blob/master/flag/flag.go#L41